src/Symfony/Component/Uid/Command/GenerateUlidCommand.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Uid\Factory\UlidFactory;
+use Symfony\Component\Uid\Ulid;
+
+class GenerateUlidCommand extends Command
+{
+    protected static $defaultName = 'ulid:generate';
+
+    private $factory;
+
+    public function __construct(UlidFactory $factory = null)
+    {
+        $this->factory = $factory ?? new UlidFactory();
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputOption('timestamp', null, InputOption::VALUE_REQUIRED, 'The ULID timestamp: a parsable date/time string by the \DateTimeImmutable constructor. It must be greater than or equals to the UNIX epoch (1970-01-01 00:00:00)'),
+                new InputOption('count', 'c', InputOption::VALUE_REQUIRED, 'The number of ULID to generate', 1),
+                new InputOption('format', 'f', InputOption::VALUE_REQUIRED, 'The ULID output format: canonical, base_58 or rfc_4122', 'canonical'),
```

Comment:
--base=16/32/58 - no need for the canonical way, as that's the default?

Reply:
In other commands with a "format" option, we include the default format in the listed possibilities.

Reply:
base32, base58, rfc4122